AdSecGH/Components/3_Section/FlattenRebar.cs

```diff
@@ -1,130 +1,139 @@
 ﻿using System;
-using System.Collections.Generic;
+using System.Drawing;
 using System.Linq;
 
-using AdSecGH.Helpers;
-using AdSecGH.Helpers.GH;
 using AdSecGH.Parameters;
+using AdSecGH.Properties;
 
 using Grasshopper.Kernel;
 
-using Oasys.AdSec;
-using Oasys.AdSec.Reinforcement.Groups;
-using Oasys.AdSec.Reinforcement.Preloads;
-using Oasys.Profiles;
+using Oasys.Business;
+using Oasys.GH.Helpers;
 
 using OasysGH;
-using OasysGH.Components;
-using OasysGH.Parameters;
 using OasysGH.Units;
 
-using OasysUnits;
+using Attribute = Oasys.Business.Attribute;
 
 namespace AdSecGH.Components {
-  public class FlattenRebar : GH_OasysComponent {
-    public override Guid ComponentGuid => new Guid("879ecac6-464d-4286-9113-c15fcf03e4e6");
-    public override GH_Exposure Exposure => GH_Exposure.tertiary | GH_Exposure.obscure;
-    public override OasysPluginInfo PluginInfo => AdSecGH.PluginInfo.Instance;
-    protected override System.Drawing.Bitmap Icon => Properties.Resources.FlattenRebar;
-
-    public FlattenRebar() : base(
-      "FlattenRebars",
-      "FRb",
-      "Flatten all rebars in a section into single bars.",
-      CategoryName.Name(),
-      SubCategoryName.Cat4()) {
-      Hidden = true;
+
+  public class FlattenRebarGhComponent : FlattenRebarComponent {
```

Comment:
here you have FlattenRebarGhComponent, and for previous file is just EditSection. Why the difference?

Reply:
could you elaborate?

Reply:
will think of something better